siriuspy/siriuspy/pwrsupply/csdev.py

```diff
@@ -1334,13 +1333,13 @@ def _get_ps_FAP_propty_database():
         'AlarmsIIBLabels-Cte': {'type': 'string',
                                 'count': len(_et.IIB_ALARMS_FAP),
                                 'value': _et.IIB_ALARMS_FAP},
-        'IIBLeakCurrent-Mon': {'type': 'float', 'value': 0.0,
+        'LeakCurrentIIB-Mon': {'type': 'float', 'value': 0.0,
                                'prec': PS_CURRENT_PRECISION,
                                'unit': 'A'},
-        'IIBInductorTemperature-Mon': {'type': 'float', 'value': 0.0,
-                                       'prec': 2,
-                                       'unit': 'C'},
-        'IIBHeatSinkTemperature-Mon': {'type': 'float', 'value': 0.0,
+        'InductorsTemperatureIIB-Mon': {'type': 'float', 'value': 0.0,
+                                        'prec': 2,
+                                        'unit': 'C'},
+        'HeatSinkTemperatureIIB-Mon': {'type': 'float', 'value': 0.0,
                                        'prec': 2,
                                        'unit': 'C'},
         'DCLinkVoltage-Mon': {'type': 'float', 'value': 0.0,
```

Comment:
Following the new standard, shouldn't it be _DCLinkVoltageIIB-Mon?_

Reply:
No, because this specific variable isn't provided b y IIB in this PS model. VoltageInputIIB is indeed.